scala/scala.bzl

```diff
@@ -579,12 +573,13 @@ def _scala_junit_test_impl(ctx):
         main_class = "org.junit.runner.JUnitCore",
         jvm_flags = launcherJvmFlags + ctx.attr.jvm_flags,
         args = test_suite.suite_class,
-        run_before_binary = "",
-        run_after_binary = "",
     )
 
     return _scala_binary_common(ctx, cjars, rjars)
 
+_launcher_template = {
+  "_java_stub_template": attr.label(default=Label("@java_stub_template//file")),
```

Comment:
I think `@java_stub...` is the right way to go here.

Also, I think it is okay that `test/BUILD` uses `@scala//:` since we aren't actually depending on tests targets remotely.